.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+    steps:
+      - checkout
+      - run:
+          name: dependancies
+          command: yarn
```

Comment:
Would you mind adding cache loading/saving steps around this command? This used to be done automatically in v1 and can drastically speed up tests. You can see how this is done in [Circle’s example Node project](https://github.com/CircleCI-Public/circleci-demo-javascript-express/blob/master/.circleci/config.yml#L13-L21) or in [web-monitoring-db’s configuration](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/.circleci/config.yml#L20-L30). *(Unlike the example project, you should make sure to include `arch` in the cache key just in case we have any compiled C extensions somewhere in the dependency tree—see the web-monitoring-db config for an example.)*

Also, small typo in “dependancies” here.